bin/node/rpc/src/lib.rs

```diff
@@ -66,7 +69,7 @@ pub struct BabeDeps {
 }
 
 /// Full client dependencies.
-pub struct FullDeps<C, P, SC> {
+pub struct FullDeps<C, P, SC, B, E, N, RA, VR> {
```

Comment:
oh wow

Reply:
Yeah I'm changing that to take `Env` directly